[2024-08-07T20:33:09.661+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T20:33:09.702+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: space_exploration.nasa_confirmation_response scheduled__2023-02-15T00:00:00+00:00 [queued]>
[2024-08-07T20:33:09.715+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: space_exploration.nasa_confirmation_response scheduled__2023-02-15T00:00:00+00:00 [queued]>
[2024-08-07T20:33:09.716+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-07T20:33:10.220+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): nasa_confirmation_response> on 2023-02-15 00:00:00+00:00
[2024-08-07T20:33:10.239+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=94) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-07T20:33:10.242+0000] {standard_task_runner.py:64} INFO - Started process 124 to run task
[2024-08-07T20:33:10.242+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'space_exploration', 'nasa_confirmation_response', 'scheduled__2023-02-15T00:00:00+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/***-space-exploration.py', '--cfg-path', '/tmp/tmpw9tjiiym']
[2024-08-07T20:33:10.251+0000] {standard_task_runner.py:91} INFO - Job 47: Subtask nasa_confirmation_response
[2024-08-07T20:33:10.351+0000] {task_command.py:426} INFO - Running <TaskInstance: space_exploration.nasa_confirmation_response scheduled__2023-02-15T00:00:00+00:00 [running]> on host 13935ade345d
[2024-08-07T20:33:10.544+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='space_exploration' AIRFLOW_CTX_TASK_ID='nasa_confirmation_response' AIRFLOW_CTX_EXECUTION_DATE='2023-02-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-02-15T00:00:00+00:00'
[2024-08-07T20:33:10.546+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T20:33:10.566+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-07T20:33:10.567+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 20 && echo "Confirmation from NASA, you can proceed." > /tmp/response_20230215.txt']
[2024-08-07T20:33:10.584+0000] {subprocess.py:86} INFO - Output:
[2024-08-07T20:33:30.587+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-07T20:33:30.589+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T20:33:30.638+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=space_exploration, task_id=nasa_confirmation_response, run_id=scheduled__2023-02-15T00:00:00+00:00, execution_date=20230215T000000, start_date=20240807T203309, end_date=20240807T203330
[2024-08-07T20:33:30.681+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-07T20:33:30.699+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
